service-per-pod/values.yaml

```diff
@@ -0,0 +1,22 @@
+service:
+  type: "LoadBalancer"
+annotations:
+  service-per-pod-label: "statefulset.kubernetes.io/pod-name"
+  service-per-pod-ports: "5701:5701"
+initContainers:
+  - name: wait-for-lb
+    image: eu.gcr.io/hazelcast-33/kubectl-jq:0.1.0
```

Comment:
I guess this image won't be available to the public, right? Wouldn't it be better to have some image that can be fetched from `docker-hub` instead?